tsl/test/expected/hypertable_generalization.out

```diff
@@ -0,0 +1,495 @@
+ Append  (cost=0.15..14.71 rows=16 width=24)
+   ->  Index Scan using "11_2_jobs_serial_pkey" on _timescaledb_internal._hyper_4_11_chunk  (cost=0.15..7.31 rows=8 width=24)
+         Output: _hyper_4_11_chunk.job_id, _hyper_4_11_chunk.device_id, _hyper_4_11_chunk.start_time, _hyper_4_11_chunk.end_time
+         Index Cond: ((_hyper_4_11_chunk.job_id >= 30) AND (_hyper_4_11_chunk.job_id < 90))
+   ->  Index Scan using "12_3_jobs_serial_pkey" on _timescaledb_internal._hyper_4_12_chunk  (cost=0.15..7.31 rows=8 width=24)
+         Output: _hyper_4_12_chunk.job_id, _hyper_4_12_chunk.device_id, _hyper_4_12_chunk.start_time, _hyper_4_12_chunk.end_time
+         Index Cond: ((_hyper_4_12_chunk.job_id >= 30) AND (_hyper_4_12_chunk.job_id < 90))
+(7 rows)
+
+EXPLAIN VERBOSE SELECT * FROM jobs_serial WHERE job_id > 90;
+                                                           QUERY PLAN                                                            
+---------------------------------------------------------------------------------------------------------------------------------
+ Append  (cost=0.15..45.84 rows=1046 width=24)
+   ->  Index Scan using "13_4_jobs_serial_pkey" on _timescaledb_internal._hyper_4_13_chunk  (cost=0.15..20.30 rows=523 width=24)
+         Output: _hyper_4_13_chunk.job_id, _hyper_4_13_chunk.device_id, _hyper_4_13_chunk.start_time, _hyper_4_13_chunk.end_time
+         Index Cond: (_hyper_4_13_chunk.job_id > 90)
+   ->  Index Scan using "14_5_jobs_serial_pkey" on _timescaledb_internal._hyper_4_14_chunk  (cost=0.15..20.30 rows=523 width=24)
+         Output: _hyper_4_14_chunk.job_id, _hyper_4_14_chunk.device_id, _hyper_4_14_chunk.start_time, _hyper_4_14_chunk.end_time
+         Index Cond: (_hyper_4_14_chunk.job_id > 90)
+(7 rows)
+
+-- Update rows
+UPDATE jobs_serial SET end_time = end_time + INTERVAL '1 hour' where job_id = 1;
+UPDATE jobs_serial SET end_time = end_time + INTERVAL '1 hour' where job_id = 30;
+UPDATE jobs_serial SET end_time = end_time + INTERVAL '1 hour' where job_id = 90;
+SELECT start_time, end_time FROM jobs_serial WHERE job_id = 1;
+          start_time          |           end_time           
+------------------------------+------------------------------
+ Fri Mar 02 01:00:00 2018 PST | Sat Mar 03 02:00:00 2018 PST
+(1 row)
+
+SELECT start_time, end_time FROM jobs_serial WHERE job_id = 30;
+          start_time          |           end_time           
+------------------------------+------------------------------
+ Sat Mar 03 06:00:00 2018 PST | Sun Mar 04 07:00:00 2018 PST
+(1 row)
+
+SELECT start_time, end_time FROM jobs_serial WHERE job_id = 90;
+          start_time          |           end_time           
+------------------------------+------------------------------
+ Mon Mar 05 18:00:00 2018 PST | Tue Mar 06 19:00:00 2018 PST
+(1 row)
+
+-- Delete rows
+DELETE FROM jobs_serial WHERE job_id < 10;
+SELECT count(*) FROM jobs_serial WHERE job_id < 30;
```

Comment:
It is hard to see from the test output that tuples (and only the intended tuples) are deleted since the previous count is not shown and different predicates are used.

```
-- Delete rows

-- Existing tuple counts
SELECT count(*) FROM jobs_serial;
SELECT count(*) FROM jobs_serial WHERE job_id < 10;

-- Perform the delete
DELETE FROM jobs_serial WHERE job_id < 10;

-- Ensure only the intended tuples are deleted
SELECT count(*) FROM jobs_serial;
SELECT count(*) FROM jobs_serial WHERE job_id < 10;
```

Reply:
Did something similar but compare before and after so that we do not have to update the result files each time.